src/components/chat-view/chat-input/ChatUserInput.tsx

```
@@ -9,22 +9,30 @@ import {
   useRef,
   useState,
 } from 'react'
+import { Notice } from 'obsidian'
```

ESLint

[error] `obsidian` import should occur before import of `react`

(import/order)